synapse/lib/cell.py

```diff
@@ -488,7 +495,14 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
         },
     }
 
-    async def __anit__(self, dirn, conf=None, readonly=False, *args, **kwargs):
+    async def __anit__(self, dirn, conf=None, readonly=False, deferpost=False, *args, **kwargs):
+        '''
+            Args:
+                deferpost(bool): Set to true if subclass is calling postNexsAnit itself
```

Comment:
If we are documenting args here, lets document them all.

Reply:
OBE